dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -179,9 +179,7 @@ def toggle_hidden
   def stage_extras
     authorize! :read, ScriptLevel
 
-    if current_user&.teacher? && !current_user&.sections&.all?(&:stage_extras)
-      flash[:info] = I18n.t(:stage_extras_teacher_message).html_safe
-    end
+    @show_stage_extras_warning = current_user&.teacher? && !current_user&.sections&.all?(&:stage_extras)
```

Comment:
could we change this logic so that we only show this warning if the section currently being viewed has stage_extras turned off? it's confusing to me to display this notification but not tell the teacher _which_ section(s) have stage_extras disabled.

this is more of a product question, so tagging @fontie715 as well

Reply:
Great point! I'll make that change and then we can confirm with @fontie715 when she's back.

Reply:
![stage-extras-specific-section](https://user-images.githubusercontent.com/12300669/64732690-1ee5b980-d498-11e9-8567-76f1f3931aeb.gif)

I changed the wording slightly to reference the current section and now only show the warning for sections that have lesson extras turned off.